librocksdb_sys/crocksdb/crocksdb/c.h

```diff
@@ -1099,6 +1099,14 @@ extern C_ROCKSDB_LIBRARY_API void crocksdb_options_set_delayed_write_rate(
 /* RateLimiter */
 extern C_ROCKSDB_LIBRARY_API crocksdb_ratelimiter_t* crocksdb_ratelimiter_create(
     int64_t rate_bytes_per_sec, int64_t refill_period_us, int32_t fairness);
+typedef enum crocksdb_ratelimiter_mode_t {
```

Comment:
We tend to define types at the beginning of this header file, can you move it there?